backend/uv-backend.py

```diff
@@ -409,6 +433,395 @@ def credit_card_details(transactionId):
+            kg_em_cmd = f"LIST KG.EM WITH @ID = '{emp_id}' LNAME FNAME COMPANY.CODE TOJSON"
+            kg_em_details = u2py.Command(kg_em_cmd).run(capture=True)
+            kg_em_details = json.loads(kg_em_details)
+            pct = kg_em_details["KG.EM"]["0"]["COMPANY.CODE"]+' employee'
+            pct = pct + \
+                ' ('+kg_em_details["KG.EM"]["0"]["FNAME"] + \
+                ' '+kg_em_details["KG.EM"]["0"]["LNAME"]+')'
+        elif(disc_type == 10):
+            pct = 'Affiliate employee family'
+            pct = kg_em_details["KG.EM"]["0"]["COMPANY.CODE"] + \
+                ' employee family'
+            pct = pct + \
+                ' (of '+kg_em_details["KG.EM"]["0"]["FNAME"] + \
+                ' '+kg_em_details["KG.EM"]["0"]["LNAME"]+')'
+        elif(disc_type == 11):
+            pct = 'Coupon [id '+coupon_code+']'
+        elif(disc_type == 12):
+            pct = 'Perfect Fit signup'
+        elif(disc_type == 13):
+            pct = ' "PROMO"'
+            pct = pct+' ('+promos_id+')'
+            if(other_sub_type == 1):
+                pct = pct+' (automatically applied)'
+            elif(other_sub_type == 2):
+                pct = pct+' (required a coupon)'
+            else:
+                pct = pct+' (?)'
+        elif(disc_type == 14):
+            pct = "N-FOR"
+            pct = pct+' ('+promos_id+')'
+        else:
+            pct = 'NO'
+        sale_total = discount
+        data = str(pct)+' discount of '+str(scr_discount*100)
+        if (disc_conv_to_mkdn):
+            data = data+' converted to markdown.'
+        else:
+            data = data+'.'
+        sale_total=0
+        items_mv=len(details['TRANSACTION'][0]['ITEM_MV'])
+        for i in range(0,items_mv):
+                retail=float(details['TRANSACTION'][0]['ITEM_MV'][count]['RETAIL'])
+                mrkdn=float(details['TRANSACTION'][0]['ITEM_MV'][count]['LONG.MRKDN'])
+                quantity=int(details['TRANSACTION'][0]['ITEM_MV'][count]['QUANTITY'])
+                sale_total=sale_total+(retail-mrkdn)*quantity
+        sale_total=sale_total+float(details['TRANSACTION'][0]["TAX.AMT"])
+        sale_total=sale_total-discount
+        discount_details['pct'] = pct
+        discount_details['subTotal'] = sale_total
+        response = {
+            "discountDetails": discount_details
+        }
+        return Response(
+            json.dumps(response),
+            status=200,
+            mimetype='application/json')
+    else:
+        msg = 'Transaction file not found'
+        data = {
+            'msg': msg
+        }
+        return Response(
+            json.dumps(data),
+            status=404,
+            mimetype='application/json'
+        )
+
+#############################################################
+####################### Refund API ##########################
+#############################################################
+
+@app.route('/transaction/refund/<transactionId>', methods=['GET'])
```

Comment:
Routes should be [RESTful](https://medium.com/@thejasonfile/restful-routing-2056f799223e)